app/javascript/components/Editor.tsx

```diff
@@ -192,7 +192,7 @@ export function Editor({
         }
 
         const iteration = typecheck<Iteration>(json, 'iteration')
-        location.assign(iteration.links.self)
+        location.assign(iteration.links.solution)
```

Comment:
We want to redirect to the solution page after the submission, not the iteration page. So I've changed that here and added it to the serializer.